newsletter/templates/newsletter/message/message.txt

```diff
@@ -12,4 +12,4 @@
 
 ++++++++++++++++++++
 
-Unsubscribe: http://{{ site }}{% url "newsletter_unsubscribe_request" newsletter.slug %}
+{% trans "Unsubscribe:" %} http://{{ site }}{% url "newsletter_unsubscribe_request" newsletter.slug %}

```

Comment:
Why would `{% trans "Unsubscribe:" %}` be better syntax than `{% trans "Unsubscribe" %}:`? Could be I'm missing linguistic subleties here.